### DISCUSSION TITLE
Why are my curves so noisy?

### DISCUSSION THREAD

**Discussion by emillundh** [ultralytics/yolov5#9412] [Q&A]
So this might be more of a general ML or object detection question... but what to do when the curves look like the attached?
I have trained on a custom dataset of about 20000 images, retrained the pretrained yolov5s, and as you can see I have only one class.
I see no overfitting, as in val curves turning downwards, instead I see a quite quick leveling off and significant jumping from epoch to epoch.
Does this look familiar to any of you? Facing this kind of curve, where would you start looking for improvements?
![noisy_loss_curves](https://user-images.githubusercontent.com/3864212/190110455-d35e9b1f-2f58-462e-bef8-28c346bdcac4.png)

**Comment by glenn-jocher** [comment]
@emillundh what noise level were you expecting?
